### DISCUSSION TITLE
Support Net Core 6.0

### DISCUSSION THREAD

**Discussion by ricardolerma** [kirov-opensource/NAutowired#33] [Ideas]
Please make available a version of this beautiful project compatible with Net Core 6.0

**Comment by FatTigerWang** [comment]
Currently able to support net6.0

**Reply by ricardolerma** [reply]
Hello, I am referring to net core 6, the input methods have been changed in the netcore 6 version, for example the Startup.cs has been removed. I tried to follow the example by configuring through Program.cs the references were not invalid. Would you be able to reproduce an example configuration for Netcore 6? Thanks.

**Reply by FatTigerWang** [reply]
I added sample for [aspnetcore6](https://github.com/kirov-opensource/NAutowired/tree/master/Sample/NAutowired.AasNetCore6.WebAPI.Sample)

**Reply by ricardolerma** [reply]
Hi @FatTigerWang, I appreciate the help, but in your example you are using the Startup.cs class, but netcore 6 doesn't use this class anymore, I didn't understand this detail.

**Reply by FatTigerWang** [reply]
I added new example for aspnetcore6, it doesn't have Startup.cs
